README.md

```diff
@@ -164,8 +164,6 @@ The below documentation will use the following pattern:
 
 - `customClass` (string) | `'nsm-dialog-animation-fade'` ― _All the additionnal classes you want to add to the modal (e.g.: any bootstrap modal class). You can add several classes by giving a string with space-separated class names_
 
-- `visible` (boolean) | `false` ― _Define if the modal is shown or not. Automatically toggled with open() and close()/dismiss() methods._
```

Comment:
Something is strange here, you remove this from the doc but the component still have this input 

`@Input() public visible: boolean = false;`

Reply:
Please read [this thread](https://github.com/biig-io/ngx-smart-modal/issues/108#issuecomment-424022698)!